### PR TITLE
Change distribution actions to use FL_CHANGELOG value by default

### PR DESCRIPTION
**Pull request by asfalcone**
if no other release notes or notes path is specified

### CODE REVIEW

File: spec/actions_specs/hockey_spec.rb [hunk 1]
```diff
@@ -62,6 +62,19 @@
         expect(values[:upload_dsym_only]).to eq(false)
       end
 
+      it "can use a   generated changelog as release notes" do
```
Comment (KrauseFx) [inline spec/actions_specs/hockey_spec.rb]: Ups, double space after `a`

File: lib/fastlane/actions/crashlytics.rb [hunk 2]
```diff
@@ -7,16 +7,16 @@ def self.run(params)
 
         params.values # to validate all inputs before looking for the ipa/apk
 
+        # We need to store notes in a file, because the crashlytics CLI (iOS) says so
         if params[:notes]
-          require 'tempfile'
-          # We need to store it in a file, because the crashlytics CLI (iOS) says so
-          Helper.log.error "Overwriting :notes_path, because you specified :notes" if params[:notes_path]
+          Helper.log.error "Overwriting :notes_path, because you have :notes" if params[:notes_path]
```
Comment (KrauseFx) [inline lib/fastlane/actions/crashlytics.rb]: Did you intend to remove `specified`?

File: lib/fastlane/actions/crashlytics.rb [hunk 3]
```diff
@@ -7,16 +7,16 @@ def self.run(params)
...
 
-          changelog = Tempfile.new('changelog')
-          changelog.write(params[:notes])
-          changelog.close
+          params[:notes_path] = Helper::CrashlyticsHelper.write_to_tempfile(params[:notes], 'changelog').path
+        elsif Actions.lane_context[SharedValues::FL_CHANGELOG] && !params[:notes_path]
+          Helper.log.info "Sending FL_CHANGELOG as release notes to Beta by Crashlytics"
```
Comment (KrauseFx) [inline lib/fastlane/actions/crashlytics.rb]: That's great :+1:

File: lib/fastlane/actions/hockey.rb [hunk 4]
```diff
@@ -88,7 +88,7 @@ def self.available_options
           FastlaneCore::ConfigItem.new(key: :notes,
                                        env_name: "FL_HOCKEY_NOTES",
                                        description: "Beta Notes",
-                                       default_value: "No changelog given"),
+                                       default_value: Actions.lane_context[SharedValues::FL_CHANGELOG] || "No changelog given"),
```
Comment (KrauseFx) [inline lib/fastlane/actions/hockey.rb]: :+1:

**Comment by KrauseFx** [comment]
I added some small comments before it's ready to merge, thank you :+1:

File: lib/fastlane/actions/crashlytics.rb (hunk 2 above)
Comment (asfalcone) [inline lib/fastlane/actions/crashlytics.rb]: Nope, fixed.

File: spec/actions_specs/hockey_spec.rb (hunk 1 above)
Comment (asfalcone) [inline spec/actions_specs/hockey_spec.rb]: Fixed.

**Comment by KrauseFx** [comment]
Yeah, that's great :rocket:

**Pull request merged**